src/Options/index.js

```diff
@@ -520,3 +523,18 @@ export interface AuthAdapter {
   */
   enabled: ?boolean;
 }
+
+export interface LogLevelUses {
+  /* Log level used by the after hook trigger, default is 'info'.
+  :DEFAULT: 'info'
+  */
+  triggerAfterHook: ?string;
+  /* Log level used by the success before hook trigger, default is 'info.
+  :DEFAULT: 'info'
+  */
+  triggerSuccessBeforeHook: ?string;
+  /* Log level used by the error before hook trigger, default is 'error.
```

Comment:
```suggestion
  /* Log level used by the Cloud Code Triggers `beforeSave`, `beforeSaveFile`, `beforeDeleteFile`, `beforeFind`, `beforeLogin` on success. Default is `error `.
```

Again, not sure whether this also covers LQ triggers.